pkg/operator/ceph/object/spec.go

```diff
@@ -509,19 +509,6 @@ func (c *clusterConfig) defaultStartupProbe() (*v1.Probe, error) {
 	return probe, nil
 }
 
-func (c *clusterConfig) generateReadinessProbeScheme() v1.URIScheme {
```

Comment:
one doubt is the function okay to remove or there might be libbucket provisionerr depending on this

Reply:
Looks like it's dead code, the readiness probe was removed recently, Blaine can give more background if needed

Reply:
Yup feel the same, mostly the API part is dependent on that.